config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
One more thing to consider. if `DEBUG` is set to true, your application can potentially leak your secret, right?
Considering that, I think you'd better use the filter.

The `DEBUG` configuration is sensitive. It should default to `false` especially in production environment in my opinion. You choose the `true` default value, probably because it's good for first-time users. That's good, but then you need more safeguard for people.

If anyone set the `DEBUG` env var to `false` (or other potential values indicates false except for `0`), the app can still run with DEBUG=true, which is very dangerous. Their secrets may be publicly available if the app has an error. You can still say RTFM to them, but it's not user friendly.